zerver/lib/bulk_create.py

```diff
@@ -63,6 +66,7 @@ def bulk_create_users(realms, users_raw, bot_type=None):
     Subscription.objects.bulk_create(subscriptions_to_create)
 
 def bulk_create_streams(realms, stream_list):
+    # type: (Dict[Any, Realm], Union[Set[Tuple[Any, Any]], List[Tuple[Any, Any]]]) -> None
```

Comment:
Same comments as above -- we shouldn't be using Any for things where the type is a string